src/Suave.DotLiquid/paket.template

```diff
@@ -13,8 +13,6 @@ files
   bin/Release/netstandard2.0/Suave.DotLiquid.pdb => lib/netstandard2.0
 dependencies
   framework: net461
-    FSharp.Core ~> 4
```

Comment:
the `FSharp.Core` is removed from paket.template, so the constraint in `paket.dependencies` will be used
that mean `>= 4.3.4 && <5`